### ISSUE TITLE
ArgumentError when using ">=" select

### ISSUE THREAD

**Issue by pablohirafuji** [blitzstudios/triton#7]
Hi, thanks for open sourcing this package, it has been really useful!

I am having problem when using ">=" in a select statement:
```elixir
Article.ByGeoHash
  |> select(:all)
  |> where(geohash: [">=": cursor])
  |> limit(10)
  |> Article.ByGeoHash.all()
```
It gives me this error:
```
** (exit) an exception was raised:
    ** (ArgumentError) cannot convert the given list to a string.

To be converted to a string, a list must contain only:

  * strings
  * integers representing Unicode codepoints
  * or a list containing one of these three elements

Please check the given list or call inspect/1 to get the list representation, got:

[>=: "6vjynkwp9wv"]

        (elixir) lib/list.ex:821: List.to_string/1
        (triton) lib/triton/validate.ex:41: Triton.Validate.coerce_fragment/2
        (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
        (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
        (triton) lib/triton/validate.ex:37: Triton.Validate.coerce/2
        (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
        (elixir) lib/enum.ex:1294: Enum."-map/2-lists^map/1-0-"/2
        (triton) lib/triton/validate.ex:5: Triton.Validate.coerce/1
        ...
```

I'm using Elixir v 1.6.4 on Windows 10.

**Comment by jonzlin95** [comment]
Try “>= “<> cursor

**Comment by pablohirafuji** [comment]
Yes, this works! Thank you! 😄

**Comment by pablohirafuji** [comment]
Hey, your solution does compiles, but it's not working properly. I tried adding some single quotes to the comparison(`[">=" <> "'0'"]`) and gives me this error:
```
(MatchError) no match of right hand side value: {:error, "line 1:89 mismatched input '0' expecting EOF (... 1 AND geohash = '>='[0]...)"}
```
I think it is making a query like this: `geohash = '>=0'`.

**Comment by jonzlin95** [comment]
Ohh may be a tweak we need to make to Triton then. Don’t think either @weixiyen or I use “> and <“ in our queries

**Comment by pablohirafuji** [comment]
That is ok. I can take a look too and make a pull request, if you guys dont mind, but I am pretty new to Elixir.

**Comment by weixiyen** [comment]
This issue is fixed in Version 0.2.0 @pablohirafuji

**Comment by pablohirafuji** [comment]
Thank you!
